grpc/Makefile

```diff
@@ -37,48 +82,6 @@ swift:
 		--swift_out=./swift-cubic \
 		cubic.proto
 
-docs: ${DOCS_OUTDIR}/autogen-doc-cubic-proto.md
-
-${GO_OUTDIR}/cubic.pb.go: cubic.proto
-	mkdir -p ${GO_OUTDIR}
-	protoc -I . -I ${PROTOINC} --go_out=plugins=grpc:"${GO_OUTDIR}" $<
-
-# grpc-gateway assumes that the gateway package lives in the same package as the
-# main proto package.  However, the gateway code is not necessary for clients,
-# and hence we move it to a separate package (gw) and the generated file needs
-# to be edited so it can use cubicpb as an external package.  There is a test in
-# go-cubic/client_test.go that verifies that this modified package still builds
-# and serves the gateway.
-${GO_OUTDIR}/gw/cubic.pb.gw.go: cubic.proto
-	mkdir -p ${GO_OUTDIR}/gw
-	protoc -I . -I ${PROTOINC} --grpc-gateway_out=logtostderr=true:${GO_OUTDIR}/gw $<
-	sed -i -E "s/package cubicpb/package gw/g" $@
-	sed -i -E "s/Package cubicpb is/Package gw is/g" $@
-	sed -i -E "s|import \(|& \"github.com/cobaltspeech/sdk-cubic/grpc/go-cubic/cubicpb\"|" $@
-	sed -i -E "s/protoReq /&cubicpb\./g" $@
-	sed -i -E "s/(protoReq )cubicpb\.([A-Za-z]+\.)/\1\2/g" $@
-	sed -i -E "s/, client /, client cubicpb./g" $@
-	sed -i -E "s/Client /, client cubicpb./g" $@
-	sed -i -E "s/[^(]*Client, runtime/cubicpb.&/" $@
-	sed -i -E "s/New[A-Za-z]*Client/cubicpb.&/" $@
-	cd $(dir $@) && go fmt $(notdir $@)
-
-${PY_OUTDIR}/cubic_pb2.py: cubic.proto
-	mkdir -p ${PY_OUTDIR}
-	python -m grpc_tools.protoc -I . -I ${PROTOINC} --python_out=${PY_OUTDIR} $<
-
-${PY_OUTDIR}/cubic_pb2_grpc.py: cubic.proto
-	mkdir -p ${PY_OUTDIR}
-	python -m grpc_tools.protoc -I . -I ${PROTOINC} --grpc_python_out=${PY_OUTDIR} $<
-
-go-test:
-	cd go-cubic && go test
-	cd go-cubic/cubicpb/gw && go test
-
-py-test:
-	cd py-cubic/cubic && python test_client.py
-	cd py-cubic/cubic && python test_tls.py
-
-${DOCS_OUTDIR}/autogen-doc-cubic-proto.md: cubic.proto doc.md.tmpl
```

Comment:
Did we no longer want these dependencies declared here?

Reply:
there was a bug in the current setup: the go files wouldn't get rebuilt if the proto file didn't change, even if the tool versions changed. 

so these changes basically force a rebuild of the autogen everytime make is run, regardless of the dependencies.